IFC4x3/Constants/c/CIRCULARARC_38N4a1WJ509ud6zzClEh4g.xml

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <DocConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CIRCULARARC_38N4a1WJ509ud6zzClEh4g" Name="CIRCULARARC" UniqueId="c85c4901-8131-4027-89c6-f7d32f3ab12a">
-	<Documentation>Circular arc transiiton.</Documentation>
+	<Documentation>In the geometric perspective, it denotes a connection between two points that follows a circular path. In the dynamic perspective, it denotes a segment with constant lateral acceleration on the moving train or vehicle, i.e. constant curvature.
```

Comment:
Maybe: "A continuous section of the circumference of a circle"?
Isn't a train a kind of vehicle?

Reply:
There has been a change suggested and accepted. Could you have a look at the new text?

On cubic, I agree. I would modify it.